### ISSUE TITLE
404 on main page

### ISSUE THREAD

**Issue by MrWindmark** [lydianlights/yet-another-factory-planner#19]
![Screenshot from 2023-08-10 19-35-15](https://github.com/lydianlights/yet-another-factory-planner/assets/28528567/085cc4c3-d34a-4fe5-9294-6f667eadfc18)
404 error on load main page. Check your server please

**Comment by MrWindmark** [comment]
![Screenshot from 2023-08-10 19-35-15](https://github.com/lydianlights/yet-another-factory-planner/assets/28528567/24c1ee01-bf2d-4190-98d9-e82a68571440)

**Comment by AcclaimedAP** [comment]
Hi! I think they are currently aware, they mentioned in the README.md they are in combination with a busy life, also looking to transferring host to a better alternative. Sadly it is my go-to planner, but hopefully it ends up coming back up soon!

**Comment by MrWindmark** [comment]
Yeah it's understandable. But Issue should be opened for proforma.
